src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -26,13 +29,46 @@ export default function VitalPercents(props: Props) {
         return (
           <VitalStatus key={p.vitalState}>
             {p.vitalState === VitalState.POOR && (
-              <IconFire color={vitalStateColors[p.vitalState] as Color} />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Poor'
+                    : tct('Poor: >[threshold]ms', {
+                        threshold: webVitalPoor[props.vital],
+                      })
+                }
+              >
+                <IconFire color={vitalStateColors[p.vitalState] as Color} />
+              </Tooltip>
             )}
             {p.vitalState === VitalState.MEH && (
-              <IconWarning color={vitalStateColors[p.vitalState] as Color} />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Needs improvement'
+                    : tct('Needs improvement: >[threshold]ms', {
+                        threshold: webVitalMeh[props.vital],
+                      })
+                }
+              >
+                <IconWarning color={vitalStateColors[p.vitalState] as Color} />
+              </Tooltip>
             )}
             {p.vitalState === VitalState.GOOD && (
-              <IconCheckmark color={vitalStateColors[p.vitalState] as Color} isCircled />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Good'
```

Comment:
Just saying "Good" here doesn't add any additional value. Don't have a better suggestion at the moment but will update if I think of anything.

Reply:
We should also wrap these strings with `t()`

Reply:
I had that impression too about "Good". I mainly wanted to show the standard words (maybe someone sees this and googles "web vitals good needs improvement poor"?), but I agree that it's not terribly useful. Agreed on `t`.